src/components/ValidateSubscriber/ValidateSubscribers.test.tsx

```diff
@@ -16,7 +16,7 @@ export const maxSubscribersData: MaxSubscribersData = {
   questionsList: [
     {
       answer: {
-        answerType: AnswerType[1],
+        answerType: `${AnswerType.TEXTFIELD}`,
```

Comment:
No alcanza con esto?

```suggestion
        answerType: AnswerType.TEXTFIELD,
```

Reply:
No quería hacer mas grande esta refactorización, ahí hay que centralizar el uso de `AnswerType `en algunas partes de usa _string_ (**"TEXTFIELD"**) y otras numérico.

Lo puedo hacer en otro PR o en este, que te parece mejor?

Reply:
Ahh, es un enum! nah, dejalo así.

Creo que en general no es una buena practica usar los enums de TypeScript ya que generan código, no son solo tipos. Pero si ya está, dejalo así.